test/tools/ossfuzz/solProto.proto

```diff
@@ -0,0 +1,108 @@
+/*
+	This file is part of solidity.
+
+	solidity is free software: you can redistribute it and/or modify
+	it under the terms of the GNU General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	solidity is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+	GNU General Public License for more details.
+
+	You should have received a copy of the GNU General Public License
+	along with solidity.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+syntax = "proto2";
+
+message InterfaceFunction {
+    enum StateMutability {
+        PURE = 0;
+        VIEW = 1;
+        PAYABLE = 2;
+    }
+    required StateMutability mut = 1;
+}
+
+message LibraryFunction {
+    // Library functions cannot be payable
+    enum StateMutability {
+        PURE = 0;
+        VIEW = 1;
+    }
+    enum Visibility {
+        PUBLIC = 0;
+        EXTERNAL = 1;
+        INTERNAL = 2;
+        PRIVATE = 3;
+    }
+    required Visibility vis = 1;
+    required StateMutability mut = 2;
+}
+
+message ContractFunction {
+    enum StateMutability {
+        PURE = 0;
+        VIEW = 1;
+        PAYABLE = 2;
```

Comment:
Nonpayable missing

Reply:
Add nonpayable to spec

Reply:
I think nonpayable is not a valid state mutability any longer: please see #8736

Reply:
Same update here. Please ignore my previous comment.